Implemented NfcV Tag check

henkeeman opened the pull request:
Implemented a way to see if the nfc tag is NfcV (ISO 15693) or not on android devices.

This is becouse we want to reverse the array if its a NfcV tag to match it with how IOS handles it.

We dont want to reverse it if its NfcA for an example. Becouse android and IOS handles the reading the same.

Commit:
Implemented Tag NfcV check

Implemented a way to see if the nfc tag is NfcV (ISO 15693) or not.

This is becouse we want to reverse what we read if its NfcV to match it with how IOS handles it.

We dont want to reverse it if its NfcA for an example becouse android and IOS handles the reading the same.

Edden01 commented on android/src/main/kotlin/it/matteocrippa/flutternfcreader/NfcHandler.kt:
```diff
@@ -100,19 +100,24 @@ private fun ByteArray.bytesToHexString(): String? {
+    // Get the ID from the tag, reverse only if it's NfcV (ISO 15693)
+    val id = if (isNfcV) {
+        // reverse array to convert to big-endian to match CoreNFC implementation of ISO 15693
+        this.id.reversedArray().bytesToHexString()
```
change this.id to this.getId()

henkeeman replied:
Fixed :)

Commit:
Changed from id to getId()

Commit:
Changed comment

By reversing the array on Android we convert it to little endian as Android reads it as big endian originally.
